src/components/Species/index.tsx

```diff
@@ -498,11 +500,13 @@ export default function SpeciesList({ reloadData, species }: SpeciesListProps):
   }, [columns, setSelectedColumns, activeLocale]);
 
   useEffect(() => {
-    if (speciesState?.checkData) {
-      setSpeciesState({ checkData: false });
+    const shouldCheckData = query.get('checkData');
```

Comment:
if `checkData` is `false` we will still check data. My suggestion is to use a query param with no value.
Here's an example: https://github.com/terraware/terraware-web/blob/main/src/components/NurseryWithdrawals/NurseryReassignment.tsx#L134

Reply:
Why will we still check data? If shouldCheckData = false ?

Reply:
false is a string not boolean